rsocket-core/src/test/java/io/rsocket/frame/RequestFlyweightTest.java

```diff
@@ -40,7 +40,7 @@ void testEncodingWithEmptyMetadata() {
 
     frame = FrameLengthFlyweight.encode(ByteBufAllocator.DEFAULT, frame.readableBytes(), frame);
 
-    assertEquals("00000e0000000119000000000100000064", ByteBufUtil.hexDump(frame));
+    assertEquals("00000b0000000118000000000164", ByteBufUtil.hexDump(frame));
```

Comment:
The value for this test in the develop branch is the correct value. Changing to this value is a bug.

Reply:
I got an issue on the decoding side in case we send `EMPTY` `bytebuf` metadata, so the decoding side seeing meta flag tries to read another one byte and gets the index out of bound exception

Reply:
yeah - I think you're pulling in older framing logic or something. The value 00000e0000000119000000000100000064 is correct

Reply:
fixed